chapters/background.tex

```diff
@@ -1,4 +1,13 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
+Since Viola and Jones proposed "Rapid Object Detection using a Boosted Cascade of Simple Features" in 2001, their concept is the basis of most face detection approaches.
```

Comment:
I think you could also write "Ever since Viola [...]". But this sentence is correct as well and it's up to your preferences whether you want to change it or not.